install/kubernetes/helm/istio/charts/kiali/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 description: Kiali is an open source project for service mesh observability, refer to https://github.com/kiali/kiali for detail.
 name: kiali
-version: 0.3.0-alpha
-appVersion: 0.3.0.Alpha
+version: v0.4.0
```

Comment:
Please remove the `v` and change to `0.4.0`, as the version request SemVer 2 standard.

https://github.com/kubernetes/helm/blob/master/docs/charts.md#charts-and-versioning

Reply:
Is this a hard requirement?

The version string with the prefix "v" was purposefully decided upon recently by the Kiali team because it follows what will soon be the new go standard (see the new vgo). As Kiali is a Go app, we are standardizing on their versioning scheme, which is a semver standard prefixed with "v" -- see https://research.swtch.com/vgo-tour

How will Helm work moving forward as it appears in the future Go is going to require the prefix "v" on their otherwise semver version strings?

Reply:
`appVersion` is the version of the backend service, `version` is the version of the chart. For now, we can keep those two same.

And also I think we should always follow the styles in https://github.com/kubernetes/charts for the format of version, in https://github.com/kubernetes/charts , all of the `appVersion` and `version` do not have prefix `v`.

Reply:
@gyliu513 
Can you review it?

I think fixed everything needed in 587d8ca.